README.md

```diff
@@ -1,4 +1,19 @@
 # OpenFDA
 ## JSON File Location
-- https://open.fda.gov/apis/drug/ndc/download/
-- Place JSON file in ./data/ folder
\ No newline at end of file
+1. https://open.fda.gov/apis/drug/ndc/download/
+2. Unzip and rename 'drug-ndc.json'
+3. Place JSON file in ./data/ folder
+
+## Run in venv
+1. Clone repo
+2. Enter repo: `cd OpenFDA`
+2. Create virtual environment: `python3 -m venv ./venv`
+3. Enter venv: `source venv/bin/activate`
+4. Install requirements: `pip3 install -r requirements.txt`
+5. Run DB loader: `python app/load_data.py`
+6. Run Cherrypy server: `python app/serve_data.py`
```

Comment:
Appears there is a bug here with the sqlite db url. These two modules are erroring out with an unable to parse `../../sqlite:///drugs.db`

<details>
    <summary>Error log</summary>

```[2020/05/23 10:09:58] p4517 {app/load_data.py:12} DEBUG - Connecting to database

Connecting to the DB[2020/05/23 10:09:58] p4517 {app/load_data.py:15} WARNING - Failed connecting to database

[2020/05/23 10:10:58] p4517 {app/load_data.py:17} ERROR - Second attempt connecting to database

Connecting to the DB[2020/05/23 10:10:58] p4517 {app/load_data.py:27} ERROR - Unhandled Exception
Traceback (most recent call last):
  File "app/load_data.py", line 13, in main
    session = connect_db()
  File "/home/yevgeny/workspace/openfda/app/load_data/connect_db.py", line 13, in main
    engine = create_engine(os.environ.get('DB_URI', '../../sqlite:///drugs.db'))
  File "/home/yevgeny/.cache/pypoetry/virtualenvs/openfda-sAsR2AaS-py3.8/lib/python3.8/site-packages/sqlalchemy/engine/__init__.py", line 479, in create_engine
    return strategy.create(*args, **kwargs)
  File "/home/yevgeny/.cache/pypoetry/virtualenvs/openfda-sAsR2AaS-py3.8/lib/python3.8/site-packages/sqlalchemy/engine/strategies.py", line 54, in create
    u = url.make_url(name_or_url)
  File "/home/yevgeny/.cache/pypoetry/virtualenvs/openfda-sAsR2AaS-py3.8/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 229, in make_url
    return _parse_rfc1738_args(name_or_url)
  File "/home/yevgeny/.cache/pypoetry/virtualenvs/openfda-sAsR2AaS-py3.8/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 290, in _parse_rfc1738_args
    raise exc.ArgumentError(
sqlalchemy.exc.ArgumentError: Could not parse rfc1738 URL from string '../../sqlite:///drugs.db'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "app/load_data.py", line 25, in <module>
    main()
  File "app/load_data.py", line 18, in main
    session = connect_db()
  File "/home/yevgeny/workspace/openfda/app/load_data/connect_db.py", line 13, in main
    engine = create_engine(os.environ.get('DB_URI', '../../sqlite:///drugs.db'))
  File "/home/yevgeny/.cache/pypoetry/virtualenvs/openfda-sAsR2AaS-py3.8/lib/python3.8/site-packages/sqlalchemy/engine/__init__.py", line 479, in create_engine
    return strategy.create(*args, **kwargs)
  File "/home/yevgeny/.cache/pypoetry/virtualenvs/openfda-sAsR2AaS-py3.8/lib/python3.8/site-packages/sqlalchemy/engine/strategies.py", line 54, in create
    u = url.make_url(name_or_url)
  File "/home/yevgeny/.cache/pypoetry/virtualenvs/openfda-sAsR2AaS-py3.8/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 229, in make_url
    return _parse_rfc1738_args(name_or_url)
  File "/home/yevgeny/.cache/pypoetry/virtualenvs/openfda-sAsR2AaS-py3.8/lib/python3.8/site-packages/sqlalchemy/engine/url.py", line 290, in _parse_rfc1738_args
    raise exc.ArgumentError(
sqlalchemy.exc.ArgumentError: Could not parse rfc1738 URL from string '../../sqlite:///drugs.db' 
```

</details>